.fastlane/Fastfile

```diff
@@ -37,7 +37,8 @@ platform :ios do
   private_lane :match_development do
     match(
       app_identifier: [
-        "com.kickstarter.kickstarter.debug"
+        "com.kickstarter.kickstarter.debug",
+        "com.kickstarter.kickstarter.debug.RichPushNotifications"
```

Comment:
TODO: Before we ship this, I need to make this change for the Enterprise and App Store targets. But I wanted to get a sign-off from either @ifosli or @scottkicks before I do it - since I'll have to run/update our fastlane certs to do so. I'm fine doing that for the dev target, but want to make sure I don't bury us in certificate problems by doing it for the other app targets!

Reply:
discussed offline and decided that we should have enough time during this sprint to address any certificate issues that come up.

Reply:
💯 Yeah my plan is to do the other upgrades on Monday so I've got a full week of (non-Holiday) runway if someone goes haywire

Reply:
Done.